src/gui/tray/activitylistmodel.h

```diff
@@ -86,10 +91,13 @@ class ActivityListModel : public QAbstractListModel
 
     Q_INVOKABLE void triggerDefaultAction(int activityIndex);
     Q_INVOKABLE void triggerAction(int activityIndex, int actionIndex);
+    Q_INVOKABLE void triggerDismiss(const int activityIndex);
```

Comment:
I would expect all of those to be slots (can be called from qml and connected to it from a signal)
and i know we already had this discussion in the past
`Q_INVOKABLE` would be relevant for me if there was a return value but here it is just less capable for no benefit (except an ugly macro in front of the method declaration)